jerry-core/debugger/debugger.c

```diff
@@ -175,7 +185,7 @@ jerry_debugger_send_backtrace (const uint8_t *recv_buffer_p) /**< pointer to the
 
   backtrace_p->type = JERRY_DEBUGGER_BACKTRACE_END;
 
-  jerry_debugger_send (sizeof (jerry_debugger_send_type_t) + message_size);
+  jerry_debugger_send (sizeof (jerry_debugger_send_type_t) + message_size + 4);
 } /* jerry_debugger_send_backtrace */
```

Comment:
Please avoid magic contants, use sizeof (uint32_t) instead of 4 (maybe you should add it to the `message_size` earlier in the code. 
Furthermore, as I see, you are only checks the  `max_frame_count` size in the `max_message_size`, which is not correct. I think you should add the `total_frames` also to calculate the maximum number of items for this type which can be transmitted in one message.